content/projects/gsoc/students.adoc

```diff
@@ -33,17 +33,17 @@ We cannot make any exception. Please:
 
 === Application steps
 
-. Check out the link:/projects/gsoc/2020/project-ideas[GSoC 2020 Project ideas]
+. Check out the link:/projects/gsoc/2022/project-ideas[GSoC 2022 Project ideas]
 . Select an interesting project idea or draft your own proposal.
 . Use the link:https://docs.google.com/document/d/1dIlPLXfLbFsvcaHFuwmH9_lSCVm9m6-SgNYTNAnSZpY/[project proposal template] to write your project proposal.
 . If you are not familiar with Jenkins, read the introductory info on the website and try using Jenkins with one of your previous projects.
-. Join the _jenkinsci-gsoc-all-public@googlegroups.com_ mailing list (link:https://groups.google.com/forum/#!forum/jenkinsci-gsoc-all-public[archives]).
+. Join the link:https://community.jenkins.io[Discourse communication channel]  
```

Comment:
Show the GSoC topics rather than just showing the top level page?
```suggestion
. Join the link:https://community.jenkins.io/c/contributing/gsoc/[Discourse communication channel]
```